src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -480,7 +480,7 @@ public function __toString()
         try {
             $content = $this->getContent();
         } catch (\LogicException $e) {
-            trigger_error($e->getMessage(), E_USER_ERROR);
+            return trigger_error($e, E_USER_ERROR);
```

Comment:
You may be did not notice that the very line (the original `trigger_error($e->getMessage(), E_USER_ERROR);`) has been introduced a few days ago. It is not in any tagged version of Symfony. And the original intent and behavior (even before this very line was introduced itself) was really to stop the execution flow at this line.